components/home/homeSections/VehicleSection.js

```diff
@@ -62,16 +62,22 @@ class VehicleSection extends Component{
         let driverQuestions = this.filterQuestionsData('driver');
 
         const VehiclesHeader = () => (
-            <CardHeader title={`${name} ${index + 1}`} />
+            <CardHeader>
+             <View >
+               <Text style={styles.cardHeadertxt}>{`${name} ${index + 1}`}</Text>
+             </View>
+             </CardHeader>
         );
 
-        // would need to refine if we want to put delete vehicle section icon next to card header
-        // const VehiclesHeader = () => (
-        //     <View style={{ flexDirection: 'row' }}>
-        //       <Text category='h6' style={styles.vehicleSectionHeader}>{`${name} ${index + 1}`}</Text>
-        //       <Icon name='minus-circle' width={25} height={25} />
-        //     </View>
-        // );
+//         would need to refine if we want to put delete vehicle section icon next to card header
+         const VehiclesEditHeader = () => (
+            <CardHeader>
+             <View >
+               <Text style={styles.cardHeadertxt}>{`${name} ${index + 1}`}</Text>
+             </View>
+             <Icon name='minus-circle' width={25} height={25} onPress={() => this.setState({beforeVehicleDelete:true})}/>
```

Comment:
can we add more spacing between Vehicle text and the minus button? It looks a bit tight.

Reply:
If we are adding text, I think we should place it in the right corner instead

Reply:
I tried it for a few minutes but was not able to get it to work. I think you are really good at text formatting, we can also move it to the next sprint for sure.

Reply:
Yeah, let's talk about this later, and I can attempt this as one of my cards for next sprint